### DISCUSSION TITLE
Add support for archiving conversations

### DISCUSSION THREAD

**Discussion by hseg** [d99kris/nchat#190] [Ideas]
A la Whatsapp's "Archiving" feature, there are groups I'd like to lurk on but not stay on top of -- they clutter the contact list.
It would be helpful if they could be hidden from view and only accessible via explicit contact book search.

**Comment by d99kris** [comment]
Hi @hseg - thanks for suggesting a feature. Assuming the underlying Telegram and WhatsApp libraries provide such functions, I think it would be a good functionality to add. Will try find time to look into this.

**Comment by d99kris** [comment]
Hi again - support for archiving (`alt/opt-a`) chats and unarchiving (by opening them from the contact list) has been added in `v5.14.17` / f44ef4c.
